### DISCUSSION TITLE
Log File Repeating with just the date

### DISCUSSION THREAD

**Discussion by rpourzia** [aqualinkd/AqualinkD#215] [Q&A] [answered]
I am getting this pattern in `var/log/aqualinkd.log` 

`May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM May-22-23 18:34:47 PM`

And it keeps going like that.  Setting the option to `INFO` or `DEBUG` does not make a differece.   The application is functioning perfectly and no errors with `sudo service aqualinkd status`.

**Comment by sfeakes** [comment, marked as answer]
That’s a feature in case you forgot the date…;).   
 I changed some of the logging in the last update update and broke that.  Just uploaded a fix to the dev branch, so if you get that it should be fixed.  If you used the aqua.sh script to install you’ll need to use force-latest option.

**Comment by rpourzia** [comment]
Thank you so much!

**Reply by sfeakes** [reply]
Glad it's working and sorry for messing that up in my haste.

**Reply by rpourzia** [reply]
No worries. I am grateful for all the amazing work you've done here.  Thanks again.
